account_check_payee/wizard/account_payment_register.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = "account.payment.register"
+
+    payment_method_code = fields.Char(related="payment_method_id.code")
+    check_payee = fields.Char(
+        string="Payee",
+        compute="_compute_check_payee",
+        required=True,
+        readonly=False,
+        copy=False,
+        store=True,
+    )
+
+    @api.depends("partner_id")
```

Comment:
```suggestion
    @api.depends("partner_id", "payment_method_code")
```
Otherwise, this method is not computed when you change payment method to "Check"

Reply:
I already update it.